README.md

```diff
@@ -39,24 +39,22 @@ and run `cd pollen_py && flit install -s --user`. You will need [`flit`][flit].
 
 #### Calyx
 
-Follow these [instructions](https://docs.calyxir.org/) to install Calyx. You must complete the [first](https://docs.calyxir.org/#installing-from-source-to-use-and-extend-calyx) and [third](https://docs.calyxir.org/#installing-the-command-line-driver) sections, installing Calyx from source, but feel free to skip the second section. Then, from the root of the Calyx repository, install the Calyx interpreter by running 
-```
-cargo build -p interp
-```
-
-The last step should be running `fud check`, which will report that some tools are unavailable. This is okay for our purposes.
-
-After completing the above, run
-```
-fud config stages.calyx.exec <full path to calyx repository>/target/debug/calyx
-fud config stages.interpreter.exec <full path to calyx repository>/target/debug/interp
-fud check
-```
+Below we show you how to build Calyx from source and set it up for our use.
+We suggest you make `calyx` and `pollen` sibling directories.
+
+1. `git clone https://github.com/cucapra/calyx.git`
+2. `cd calyx`
+3. `cargo build`.
+3. `flit -f fud/pyproject.toml install -s --deps production`
+4. `fud config --create global.root <full path to Calyx repository>`
+5. `cargo build -p interp`
+6. `fud config stages.calyx.exec <full path to Calyx repository>/target/debug/calyx`
+7. `fud config stages.interpreter.exec <full path to Calyx repository>/target/debug/interp`
```

Comment:
Are these necessary? I thought they got configured this way automatically.

(If they are, then the `$(pwd)` trick could work here again.)

Reply:
Ah, so I had the same thought! I tried `fud check` on a fresh copy of Calyx just before running these two steps. The stages exist, but are accompanied by a warning that they aren't globally available and won't work from any which directory. Those warnings go away after these two steps.

Reply:
OK, sounds good! It would be nice for fud to figure this out itself someday, since the `global.root` config should theoretically suffice.